### DISCUSSION TITLE
very nice binding~  ❤

### DISCUSSION THREAD

**Discussion by jcyuan** [amerkoleci/Alimer.Bindings.WebGPU#7] [General]
compared with Silk.NET, and personally feel this lib got much more elegant and nicer style.
thanks @amerkoleci 
and it's matched with the new newest version, great!

**Comment by amerkoleci** [comment]
> compared with Silk.NET, and personally feel this lib got much more elegant and nicer style. thanks @amerkoleci and it's matched with the new newest version, great!

Thanks for kind words, feel free to spread word and become sponsor if capable.

Cheers!
